nginx/config.go

```diff
@@ -145,6 +145,7 @@ http {
 		server_name {{ if contains "." $domain }}{{ $domain }}{{ else if ne $routerConfig.PlatformDomain "" }}{{ $domain }}.{{ $routerConfig.PlatformDomain }}{{ else }}~^{{ $domain }}\.(?<domain>.+)${{ end }};
 		server_name_in_redirect off;
 		port_in_redirect off;
+		set $app_name "{{ $appConfig.Name  }}";
```

Comment:
Non-blocking nit... there appears to be two spaces before the closing curlies here.